mondey_backend/src/mondey_backend/models/questions.py

```diff
@@ -69,6 +72,7 @@ class UserQuestionPublic(QuestionPublic):
 
 class UserQuestionAdmin(QuestionAdmin):
     text: dict[str, UserQuestionText] = {}
+    visibility: bool = False
```

Comment:
this inherits from QuestionAdmin which inherits from Question which already has the visibility member
```suggestion
```